test/unit/mem/view/src/ViewSubViewTest.cpp

```diff
@@ -159,6 +161,46 @@ static auto createVecFromIndexedFn()
 }
 
 //-----------------------------------------------------------------------------
+//! Prints all elements of the buffer.
+//-----------------------------------------------------------------------------
+struct PrintBufferKernel {
+    template <typename TAcc,
```

Comment:
Could you please begin with the first type indented on a new line? Furthermore, alpaka does not currently use a space before the `<`.